### ISSUE TITLE
Links that open a page with particular contest

### ISSUE THREAD

**Issue by matous-volf** [delta-cs/seminar-domjudge#30]
be able to send a link to any page - the link also contains a contest id. after opening the link, the page should open with the contest context selected.

**Comment by zapotocnylubos** [comment]
```
https://seminar.zapotocnylubos.com/public/problems?contest_id=12
```
?contest_id=12 povinny parametr 

-> kdyz neni, vyplnit a presmerovat na url s timto parametrem
-> pokud je, nastavit contest (prepnout) a pokracovat (inspirace v change-contest route)

```php
 $this->dj->setCookie('domjudge_cid', (string)$contestId, 0, null, '', false, false,
                                                 $response)
```

**Comment by matous-volf** [comment]
resolved by #34
